server/utils/writeResponse.ts

```diff
@@ -0,0 +1,18 @@
+import { Response } from "express";
+
+interface ResponseData {
+  statusCode?: number;
```

Comment:
`statusCode` this variable name is not good. i don't know what is it.  `httpStatus` is better imo or `httpStatusCode`